### DISCUSSION TITLE
Reactive List with custom Card displaying to same row

### DISCUSSION THREAD

**Discussion by justgfather** [appbaseio/reactivesearch#1581] [Q&A]
I am trying to put my card on the same row (horizontally) but it is not working as you can see on the picture 
![Screen Shot 2020-12-23 at 9 08 24 PM](https://user-images.githubusercontent.com/22310057/103050764-1e751300-4563-11eb-9e15-8ec7625d8063.png)
Here is my code: 
`    <ReactiveList
            componentId="SearchResult"
            dataField="original_title"
            size={3}
            pagination
            react={{
              and: [
                'mainSearch',
                'ratingsFilter',
                'publishFilter',
                'authorFilter',
              ],
            }}
            renderItem={(res) => (
              <Grid item key={res.id} md={4} xs={12}>
                <BookCard book={res} />
              </Grid>
            )}
          />`

There use to be OnAllData, but i do not have it in v3.

**Comment by siddharthlatest** [comment]
@justgfather Your code doesn't show how you're arranging the filters. If you want to display the results in the same row as filters, you need to check on the arrangement of all the items.

onAllData is renamed to `onData` starting v3.
